### ISSUE TITLE
Many errors when running tests

### ISSUE THREAD

**Issue by copenat** [wso2-ballerina/module-jira#51]
**Description:**
Run the tests.bal and many errors since this module has been changed for 0.990.0

2018-12-12 14:05:06,534 INFO  [wso2/jira7] - ACTION : createProjectCategory()
2018-12-12 14:05:06,783 INFO  [wso2/jira7] - ACTION : getAllProjectSummaries()
2018-12-12 14:05:06,798 DEBUG [ballerina/http] - Cached response not found for: 'GET /project?expand=description'
2018-12-12 14:05:06,799 DEBUG [ballerina/http] - Sending new request to: /project?expand=description
2018-12-12 14:05:07,292 INFO  [wso2/jira7] - ACTION : getAllProjectCategories()
2018-12-12 14:05:07,294 DEBUG [ballerina/http] - Cached response not found for: 'GET /projectCategory'
2018-12-12 14:05:07,295 DEBUG [ballerina/http] - Sending new request to: /projectCategory
        [fail] test_createProjectCategory:
            error: {ballerina}KeyNotFound {"message":"cannot find key 'message'"}
                at wso2/jira7:0.8.15:test_createProjectCategory(tests/tests.bal:384)

        [fail] test_getAllProjectSummaries:
            error: status404:Not Found - null {}
                at ballerina/test:createBallerinaError(assert.bal:42)
                ballerina/test:assertFail(assert.bal:98)
                wso2/jira7:0.8.15:test_getAllProjectSummaries(tests/tests.bal:43)

        [fail] test_getAllProjectCategories:
            error: status404:Not Found - null {}
                at ballerina/test:createBallerinaError(assert.bal:42)
                ballerina/test:assertFail(assert.bal:98)
                wso2/jira7:0.8.15:test_getAllProjectCategories(tests/tests.bal:367)

        0 passing
        3 failing
        25 skipped

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**
Ballerina 0.990.0

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**
Run tests.bal

**Related Issues:**

**Comment by copenat** [comment]
The Client type needs extra code in the constructor to append to the URL. Something like:
```
    public function __init(JiraConfiguration jiraConfig) {
        jiraConfig.baseUrl += JIRA_REST_API_RESOURCE + JIRA_REST_API_VERSION;
        jiraConfig.clientConfig.chunking = "NEVER";
        self.jiraClient = new(jiraConfig.baseUrl, config = jiraConfig.clientConfig);
    }
```

**Comment by kalaiyarasiganeshalingam** [comment]
Fixed by PR #52
